docs/source/graphviz.rst

```diff
@@ -0,0 +1,44 @@
+Visualize task graphs
+---------------------
+
+.. currentmodule:: dask
+
+.. autosummary::
+   visualize
+
+Before executing your computation you might consider visualizing the underlying task graph.
+By looking at the inter-connectedness of tasks
+you can learn more about potential bottlenecks
+where parallelism may not be possile,
+or areas where many tasks depend on each other,
+which may cause a great deal of communication.
```

Comment:
Not-important-nit: line lengths/wraps vary a lot in this PR. When I write docs in vim I often run `gqap` per paragraph to nicely wrap to the 80 char line limit. Other editors probably can do this as well. I (personally) find it easier to read the rst docs when consistent formatting is applied to the source. Not a big deal though, no need to change anything.

Reply:
Ah, I started doing this in my personal writing and have been enjoying it.  Happy to revert though, it's not standard in this codebase.  http://rhodesmill.org/brandon/2012/one-sentence-per-line/

Reply:
Ah, interesting. No strong thoughts either way. Up to you.